vmas/simulator/environment/gym.py

```diff
@@ -17,18 +17,25 @@ class GymWrapper(gym.Env):
     def __init__(
         self,
         env: Environment,
+        return_numpy: bool = False,
+        **kwargs,
```

Comment:
```suggestion
```

Here and in the other wrappers it is better to consume all args so that it results in error if users pass the wrong args